Resources/Prototypes/Catalog/Fills/Crates/salvage.yml

```diff
@@ -88,24 +88,251 @@
         prob: 0.01
         #  - Pins
```

Comment:
This comments makes no sense now, does it?

Reply:
Fair.